quickstart/101-vm-auto-shutdown/outputs.tf

```diff
@@ -0,0 +1,64 @@
+output "resource_group_name" {
```

Comment:
Note: You need to output only the following:
- any value that is dynamically generated (e.g., rg name)
- any value needed to perform a step after the application of the execution plan (so that you can use "terraform output" to output the value to a var to be used with a subsequent azCLI/azPS command).

Reply:
Thanks. I revised the outputs.tf in commit: https://github.com/Azure/terraform/pull/255/commits/e13fcdbffaaae9092854ae80a9088659d05a6e73